src/ken/rebar.config.script

```diff
@@ -11,7 +11,12 @@
 % the License.
 
 HaveDreyfus = element(1, file:list_dir("../dreyfus")) == ok.
-HaveHastings = element(1, file:list_dir("../hastings")) == ok.
+
+HastingsHome = case os:getenv("HASTINGS_HOME") of
```

Comment:
`HastingsHome = os:getenv("HASTINGS_HOME", "../hastings")` would be cleaner imo.